commons/src/test/java/fr/insideapp/sonarqube/apple/commons/issues/MobSFScanSensorTest.java

```diff
@@ -40,6 +40,11 @@ public String nameSuffix() {
             public String language() {
                 return LANG;
             }
+
+            @Override
+            public ReportParser reportParser() {
+                return new MobSFScanReportParser(new String[]{"swift"});
```

Comment:
Since `MobSFScanSensor` is not tied to Swift files, maybe this should use a mock value such as language or even no value.

```
private final String[] EXT = {"EXT"};
```